README.md

```diff
@@ -72,6 +73,8 @@ Assist Adnan by writing a program, using any allowed language, to find the city
 
 ### INPUT
 
+You can find an example of the input file [here](./input.txt). This file contains 1 million line to run your tests and validate your solution.
```

Comment:
Also, would've been nice to give people the expected output as well.

Reply:
Once we agreed on the program I can run it and share the output for the 1M rows